packages/components/autocomplete/src/autocomplete.ts

```diff
@@ -29,18 +29,30 @@ export type AutocompleteFetchSuggestions =
   | AutocompleteData
 
 export const autocompleteProps = buildProps({
+  /**
+   * @description 1
```

Comment:
I think the description of that part was entered incorrectly. "1". The same goes for other parts